snapcraft/plugins/rust.py

```diff
@@ -76,9 +77,6 @@ def get_build_properties(cls):
     def __init__(self, name, options, project):
         super().__init__(name, options, project)
         self.build_packages.extend([
```

Comment:
I think we need the C cross toolchain here and the cross compiled std crates no?
Also I think @ChrisMacNaughton  did you need these libs for the alacrity snap you did awhile back?